logging/src/main/java/org/apache/rocketmq/logging/InternalLoggerFactory.java

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
It seems that , rocketmq's module including client,broker,namesrv, dose not adopt this InternalLoggerFactory.
When and how this new module is integrated with the other modules.

Reply:
This is Logging module,In the next PR,Slf4j api used in broker, common,remoting and other modules will be changed to InternalLoggerFactory